modulemd/tests/test-modulemd-module.c

```diff
@@ -490,6 +490,125 @@ modulemd_test_remove_streams (void)
 }
 
 
+static void
+module_test_search_streams_by_glob (void)
+{
+  g_autoptr (ModulemdModuleIndex) index = modulemd_module_index_new ();
+  g_autoptr (GError) error = NULL;
+  g_autoptr (GPtrArray) failures = NULL;
+  g_autoptr (GPtrArray) streams = NULL;
+  g_autofree gchar *yaml_path = NULL;
+  ModulemdModule *module = NULL;
+
+  yaml_path = g_strdup_printf ("%s/search_streams/search_streams.yaml",
+                               g_getenv ("TEST_DATA_PATH"));
+
+  modulemd_module_index_update_from_file (
+    index, yaml_path, TRUE, &failures, &error);
+  g_assert_no_error (error);
+
+  module = modulemd_module_index_get_module (index, "nodejs");
+  g_assert_nonnull (module);
+
+  streams =
+    modulemd_module_search_streams_by_glob (module, NULL, NULL, NULL, NULL);
+
+  g_assert_nonnull (streams);
+  g_assert_cmpint (streams->len, ==, 3);
+  g_clear_pointer (&streams, g_ptr_array_unref);
+
```

Comment:
Extraneous blank line here.

Additionally, it might be worth adding a few additional calls to `modulemd_module_search_streams_by_glob()`  where the parameters are patterns. Or just call the coverage in the other test functions good enough?

Reply:
I think the coverage in the other functions is enough, but if you think they could use some more tests we could open a "good first issue" for it.